# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dpsearch/gcc4.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- dpsearch-4.46/src/db.c.vanilla	2007-05-14 12:19:07.000000000 +0000
+++ dpsearch-4.46/src/db.c	2007-05-14 12:19:25.000000000 +0000
@@ -948,6 +948,7 @@
 #ifdef HAVE_SQL
 		  DPS_FREE(db->where);
 #endif
+		;
 	    }
 	  }
 	  for (i = dbfrom; i < dbto; i++) {
